Resources/Prototypes/Datasets/Names/last.yml

```diff
@@ -219,7 +219,6 @@
   - Howe
   - Huey
   - Hughes
-  - Hujsak
```

Comment:
🤭

Reply:
insert muh 60 people on the planet have it as a last name cope here